README.md

```diff
@@ -149,6 +149,42 @@ webfont({
   });
 ```
 
+### `beforeFontGenerationGlyphTransformation`
+
+Type: `function`
+Default: `(glyph) => { return glyph }`
+
+If you need transform glyph before transferred in to any font, you can use this option with glyphs metadata object.
+
+Example:
+
+```js
+import webfont from "webfont";
+
+const historyMap = new Map(...filled);
+
+webfont({
+  files: "src/svg-icons/**/*.svg",
+  beforeFontGenerationGlyphTransformation: obj => {
```

Comment:
I think we have bug in logic, we already have `glyphTransformFn` function for this, but it is applied only for templates, i think we move this above, it is allow to modify any glyph data, what do you think?

Reply:
yeah sounds good! I'll change it.